variants/STM32G0xx/G030F6P/generic_clock.c

```diff
@@ -20,8 +20,41 @@
   */
 WEAK void SystemClock_Config(void)
 {
-  /* SystemClock_Config can be generated by STM32CubeMX */
-#warning "SystemClock_Config() is empty. Default clock at reset is used."
+    RCC_OscInitTypeDef RCC_OscInitStruct = {0};
+  RCC_ClkInitTypeDef RCC_ClkInitStruct = {0};
```

Comment:
With compiler we used, assignation to 0 on such heterogenous structure causes compiler warning, 
(and are not required) 
```suggestion
    RCC_OscInitTypeDef RCC_OscInitStruct = {};
  RCC_ClkInitTypeDef RCC_ClkInitStruct = {};
```